nixos/modules/services/cluster/k3s/default.nix

```diff
@@ -498,7 +525,7 @@ in
           "network-online.target"
         ];
         wantedBy = [ "multi-user.target" ];
-        path = lib.optional config.boot.zfs.enabled config.boot.zfs.package;
+        path = [ pkgs.nftables ] ++ lib.optional config.boot.zfs.enabled config.boot.zfs.package;
```

Comment:
Other packages that k3s needs are wrapped with the binaries. We should add `nftables` also there to be consistent with other dependencies.

See https://github.com/NixOS/nixpkgs/blob/1d7a088c34ef229519eafd6bee5d5d70a6b9df18/pkgs/applications/networking/cluster/k3s/builder.nix#L361